go.mod

```diff
@@ -10,15 +10,16 @@ require (
 	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7
 	github.com/docker/go-connections v0.4.0
 	github.com/golang/mock v1.3.1
-	github.com/google/go-cmp v0.3.0
+	github.com/golangci/golangci-lint v1.23.6 // indirect
```

Comment:
This shouldn't be here. It should only be a dep in the context of `tools/`. See `Makefile`.